app/src/main/java/com/example/parkingsystemkotlin/mvp/model/ParkingSpaceReservationModel.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.parkingsystemkotlin.mvp.model
+
+import com.example.parkingsystemkotlin.mvp.contracts.ParkingSpaceReservationContract
+import com.example.parkingsystemkotlin.utils.DateUtils
+import java.text.SimpleDateFormat
+import java.util.*
+
+class ParkingSpaceReservationModel : ParkingSpaceReservationContract.ParkingSpaceReservationModel {
+    private var isDateStartButtonPressed = false
+    private var dateStart: Calendar = Calendar.getInstance()
+    private var timeStart: Calendar = Calendar.getInstance()
+    private var dateEnd: Calendar = Calendar.getInstance()
+    private var timeEnd: Calendar = Calendar.getInstance()
```

Comment:
it is okay to initialize all these calendars here? if the user just put save without setting the date and times the local info will be saved with the actual date/time

Reply:
Done!